src/dist/replication/replication.thrift

```diff
@@ -678,7 +678,10 @@ struct duplication_entry
     4:i64                  create_ts;
 
     // partition_index => confirmed decree
-    5:map<i32, i64>        progress;
+    5:optional map<i32, i64> progress;
```

Comment:
How about rename to `confirmed_decree`, and `pending ` rename to `pending_mutations ` , it would be more clearly.

Reply:
I think the meaning of 'pending' is too long to use variable name to describe.
If we want to name it completely, its name may be "non_confirmed_mutations_count".
So I think it's fine to use a shorter name but with some documents describing it in detail.

Reply:
Or I can rename `pending` to `not_confirmed`.

Reply:
It's OK to use `not_confirmed ` as a internal variable name, but not suitable for user faced name, such as http interface, or metric name, I might consider it's not confirmed K-V pairs, decrees, etc. I suggest you to make it more clearly for http interface, at least.

Reply:
ok, so which do you think the suitable, 'not_confirmed_mutations' or 'not_confirmed_mutations_num'?